index.html

```diff
@@ -354,19 +349,21 @@ <h4>Follow Us</h4>
             </div>
           </div>
         </div>
-        <div>
+        <div class="footer-right-col">
           <div class="footer-info">
-            <div class="copyright-text">
-              <small>support@xypo.com</small><br>
-              <small>copyright 2020 ritesh</small>
+            <div class="copyright-text">             

```

Comment:
Can you remove this unused div and check once again.

Reply:
> Can you remove this unused div and check once again.

Yes, okay